actions/Home.php

```diff
@@ -1,93 +0,0 @@
-<?php
-
-/**
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; under version 2
- * of the License (non-upgradable).
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
- *
- * Copyright (c) 2014-2018 (original work) Open Assessment Technologies SA;
- *
- *
- */
-
-namespace oat\taoCe\actions;
-
-use oat\tao\model\menu\MenuService;
-use tao_models_classes_accessControl_AclProxy;
-use oat\tao\helpers\TaoCe;
-use oat\tao\model\menu\Perspective;
-
-/**
- * The Home controller provides actions for the Home screen of the Community Edition
- *
- * @author Bertrand Chevrier <bertrand@taotesting.com>
- * @package taoCe
-
- * @license GPL-2.0
- *
- */
-class Home extends \tao_actions_CommonModule
-{
-    /**
-     * This action renders the template used by the splash screen popup
-     */
-    public function splash()
```

Comment:
To avoid breaking change we should delegate those methods to new action and deprecate them.